test/blas_test.hpp

```diff
@@ -55,10 +55,8 @@ extern Args args;
 
 using namespace blas;
 
-// The executor type used in tests
-using test_executor_t =
-    blas::Executor<blas::PolicyHandler<blas::codeplay_policy>>;
-
+// The sycl blas handle type used in tests
+using test_sb_handle_t = blas::SB_Handle;
```

Comment:
I think that typename is not needed anymore now that we can have only one handle type.